### PR TITLE
UI Fixes & ReadMe

### PR DESCRIPTION
**Pull request by Bentheminernz**
- Fixes page background & favourite colour requires sponsor message
Screenshot 2024-09-24 at 9 08 16 PM

Screenshot 2024-09-24 at 8 00 00 PM

- Footer sticks to the bottom of all pages
- Moves request data export button
Screenshot 2024-09-24 at 8 04 29 PM

- Updates README, includes instructions on how to run and contribute to the codebase

- Changes to font sizing and spacing to make certain pages not look like a dense wall of text

### CODE REVIEW

File: users/templates/users/includes/user-link.html
Comment (catgirlinspace) [inline users/templates/users/includes/user-link.html:1]: Am curious what the reasoning is behind the `if user.username` check. username should never be `None`, so this seems unnecessary to me?

**Comment by catgirlinspace** [comment]
hi, thanks so much for your pull request! i've reviewed your changes and made a few of my own (mostly in the readme and the settings page that we discussed on discord) and left a review comment asking for clarification on the user-link include. other than that one comment this looks great, thank you so much!

File: users/templates/users/includes/user-link.html
Comment (Bentheminernz) [reply users/templates/users/includes/user-link.html:1]: Fixed :) I am not sure why I added that check...

**Review by Bentheminernz — COMMENTED** [review]
Fixed feedback

**Pull request merged**